package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "w3c-css-validator",
-	"version": "1.2.0",
+	"version": "1.2.1",
```

We haven't yet published v1.2.0, you can leave the version where it's at. This bug fix will be a part of that release.

```suggestion
	"version": "1.2.0",
```